src/main/java/teammates/ui/webapi/output/InstructorData.java

```diff
@@ -1,31 +1,43 @@
 package teammates.ui.webapi.output;
 
+import teammates.common.datatransfer.InstructorPrivileges;
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 
 /**
- * The API output format of an instructor.
+ * The API output format of {@link InstructorAttributes}.
  */
 public class InstructorData extends ApiOutput {
+    private String googleId;
+    private final String name;
     private final String courseId;
     private final String email;
-
+    private String role;
     private Boolean isDisplayedToStudents;
-    private final String displayedToStudentsAs;
-    private final String name;
+    private final String displayedName;
+    private InstructorPrivileges privileges;
```

Comment:
@joanneong Check all the places where this DTO is used as the output. In those places, if any of the information is not supposed to be visible, you should handle accordingly.

Reply:
Can you clarify what you mean here? I don't think I fully understand what you are saying. Do you mean the `InstructorData` DTO?

Reply:
Yes. Remember, API responses can be traced even though the web page can decide to filter some information.